src/assisted_test_infra/test_infra/controllers/node_controllers/libvirt_controller.py

```diff
@@ -675,3 +676,48 @@ def set_single_node_ip(self, ip):
 
     def get_day2_static_network_data(self):
         pass
+
+
+def collect_virsh_logs(nodes, log_dir_name):
+    log.info("Collecting virsh logs\n")
+    os.makedirs(log_dir_name, exist_ok=True)
+    virsh_log_path = os.path.join(log_dir_name, "libvirt_logs")
+    os.makedirs(virsh_log_path, exist_ok=False)
+
+    libvirt_list_path = os.path.join(virsh_log_path, "virsh_list")
```

Comment:
Hi @eliorerz  , We use libvirt CLI directly instead python api in different places , maybe we can add the remote_libvirt host ... because we do support remore_libvirt
The assumption that we run it locally  (run_command or libvirt) may be changed in the future


> class LibvirtController(NodeController, ABC):
>     TEST_DISKS_PREFIX = "ua-TestInfraDisk"
> 
>     def __init__(
>         self,
>         config: BaseNodesConfig,
>         entity_config: Union[BaseClusterConfig, BaseInfraEnvConfig],
>         libvirt_uri: str = consts.DEFAULT_LIBVIRT_URI,
>     ):

Reply:
Ok, you can create a ticket for that.
This PR just refactored and moved things from one place to another.